src/ctim/schemas/common.clj

```diff
@@ -305,22 +348,22 @@
    "Wrote_To" "Specifies that this object wrote to the related object."
    "Created" "Specifies that this object created the related object."})
 
-(s/defschema RelationType
-  (apply s/enum (keys relations-map)))
+(s/defschema ObservableRelationType
+  (apply s/enum (keys observable-relations-map)))
 
-(s/defschema Relation
+(s/defschema ObservableRelation
   {:id s/Num
    :timestamp Time
    :origin s/Str
    (s/optional-key :origin_uri) URI
-   :relation RelationType
+   :relation ObservableRelationType
    (s/optional-key :relation_info) {s/Keyword s/Any}
```

Comment:
Yes!  I think we should be doing that for sure.  Keys are always keywords or strings.  And we should define a schema that covers accepted values as well.